src/goInstallTools.ts

```diff
@@ -121,7 +123,8 @@ function getTools(goVersion: SemVersion): string[] {
 	}
 
 	if (goConfig['useLanguageServer']) {
-		tools.push('go-langserver');
+		const languageServer = getToolFromToolPath(getLanguageServerToolPath());
+		tools.push(languageServer === 'go-langserver' ? 'go-langserver' : 'gopls');
 	}
```

Comment:
not sure I understant this condition. Why not push directly what we get from `getToolFromToolpath` . it seems if users specify something else than`go-langserver` we use `gopls` instead of what they specifeid ?

Reply:
Only because we don't uet officially support any other language servers than go-langserver and gopls